tools/testing/gatekeeper/required-tests.yaml

```diff
@@ -18,25 +24,173 @@ paths:
   static:
     # Checks that static checks are passing
-    regexps: "Static checks.*"
+    names:
+      # static-checks.yaml (build-checks.yaml)
+      - Static checks / build-checks / check (agent-ctl, make check)
+      - Static checks / build-checks / check (agent-ctl, make test)
+      - Static checks / build-checks / check (agent-ctl, make vendor)
+      - Static checks / build-checks / check (agent-ctl, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (agent, make check)
+      - Static checks / build-checks / check (agent, make test)
+      - Static checks / build-checks / check (agent, make vendor)
+      - Static checks / build-checks / check (agent, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (dragonball, make check)
+      - Static checks / build-checks / check (dragonball, make test)
+      - Static checks / build-checks / check (dragonball, make vendor)
+      - Static checks / build-checks / check (dragonball, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (genpolicy, make check)
+      - Static checks / build-checks / check (genpolicy, make test)
+      - Static checks / build-checks / check (genpolicy, make vendor)
+      - Static checks / build-checks / check (genpolicy, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (kata-ctl, make check)
+      - Static checks / build-checks / check (kata-ctl, make test)
+      - Static checks / build-checks / check (kata-ctl, make vendor)
+      - Static checks / build-checks / check (kata-ctl, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (runk, make check)
+      - Static checks / build-checks / check (runk, make vendor)
+      - Static checks / build-checks / check (runtime-rs, make check)
+      - Static checks / build-checks / check (runtime-rs, make test)
+      - Static checks / build-checks / check (runtime-rs, make vendor)
+      - Static checks / build-checks / check (runtime-rs, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (runtime, make check)
+      - Static checks / build-checks / check (runtime, make test)
+      - Static checks / build-checks / check (runtime, make vendor)
+      - Static checks / build-checks / check (runtime, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks / check (trace-forwarder, make check)
+      - Static checks / build-checks / check (trace-forwarder, make test)
+      - Static checks / build-checks / check (trace-forwarder, make vendor)
+      - Static checks / build-checks / check (trace-forwarder, sudo -E PATH="$PATH" make test)
+      - Static checks / build-checks-depending-on-kvm (runtime-rs)
+      - Static checks / check-kernel-config-version
+      - Static checks / static-checks (make static-checks)
+      # static-checks-self-hosted.yaml
+      - Static checks self-hosted / build-checks (s390x) / check (agent-ctl, make test)
```

Comment:
Hi @ldoktor ! Thanks for working on this.

All these checks are currently `Waiting for status to be reported`, likely because of

```
`Static checks self-hosted / build-checks (pull_request) Skipped`
```

Any idea why the self-hosted build checks are skipped ?